store/v2/storage/sqlite/db.go

```diff
@@ -329,6 +327,25 @@ func (db *Database) PrintRowsDebug() {
 	fmt.Println(strings.TrimSpace(sb.String()))
 }
 
+func (db *Database) executeTx(stmt string, args ...interface{}) (err error) {
```

Comment:
personal preference: It is called by one place only. You can inline the method or add some docs please that it is used for a single operation TX only. should be avoided for atomic behaviour on multiple statements